list2/ex45/index.js

```diff
@@ -0,0 +1,10 @@
+export function countingStringOccurrencesInAnArray() {
+  const count = {};
+  const fruits = ["banana", "apple", "grape", "banana", "banana", "grape", "watermelon"];
+
+  fruits.forEach((value) => {
+    count[value] = (count[value] || 0) + 1;
+  });
+  console.log(JSON.stringify(count));
+}
+countingStringOccurrencesInAnArray();
```

Comment:
Your solution correctly counts the occurrences of each string in the array and stores the results in an object. However, as mentioned in other exercises you could use a more modern approach and solve it with Array methods such as reduce.

```js
const count = fruits.reduce((acc, value) => {
    acc[value] = (acc[value] || 0) + 1;
    return acc;
  }, {});
```